src/Commands/Permission.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace Althinect\FilamentSpatieRolesPermissions\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Str;
+use Illuminate\Support\Facades\File;
+use Spatie\Permission\Models\Permission as PermissionModel;
+use Illuminate\Support\Facades\DB;
+
+class Permission extends Command
+{
+    public $config;
+
+    public array $permissions = [];
+
+    protected $signature = 'permissions:sync {--C|clean}';
+
+    protected $description = 'Generates permissions through Models or Filament Resources and custom permissions';
+
+    public function __construct()
+    {
+        parent::__construct();
+        $this->config = config('filament-spatie-roles-permissions.generator');
```

Comment:
True, since the old config file doesn't have the "generator" attribute defined.
I mentioned the updating process of the package to publish the config file. Its better to handle it within the code itself